spec/metanorma/cli/collection_spec.rb

```diff
@@ -1,31 +1,32 @@
+RESULTS = "spec/results".freeze
+
 RSpec.describe "Collection" do
+  around(:each) do |example|
+    Dir.mktmpdir("rspec-") do |dir|
+      FileUtils.cp Dir.glob("spec/fixtures/*"), dir
+      Dir.chdir(dir) { example.run }
+    end
+  end
+
   it "render HTML from YAML" do
-    of = "spec/results"
     Metanorma::Cli.start [
-      "collection", "spec/fixtures/collection1.yml", "-x", "html",
-      "-w", of, "-c", "spec/fixtures/collection_cover.html",
-      "--no-install-fonts"
+      "collection", "collection1.yml", "-x", "html", "-w", RESULTS, "-c", "collection_cover.html", "--no-install-fonts"
     ]
-    expect(File.exist?("spec/results/index.html")).to be true
-    expect(File.exist?("spec/results/dummy.html")).to be true
-    expect(File.exist?("spec/results/rice-amd.final.html")).to be true
-    expect(File.exist?("spec/results/rice-en.final.html")).to be true
-    expect(File.exist?("spec/results/rice1-en.final.html")).to be true
-    FileUtils.rm_rf of
+    expect(File.exist?("#{RESULTS}/index.html")).to be true
+    expect(File.exist?("#{RESULTS}/dummy.html")).to be true
+    expect(File.exist?("#{RESULTS}/rice-amd.final.html")).to be true
+    expect(File.exist?("#{RESULTS}/rice-en.final.html")).to be true
+    expect(File.exist?("#{RESULTS}/rice1-en.final.html")).to be true
```

Comment:
Looks like we are doing the same thing in couple of places, what do you think about having some helper method here? maybe something like

```ruby
#usages:
expect(spec_result_file_exists?("rice-en.final.html")).to be true

# helper method
def spec_result_file_exists?(filename)
  file_path = Pathname.new(RESULTS).join(filename)
  File.exist?(file_path)
end
```

Reply:
Done